commit 4c33cb1f426a1c682cb973386cb3cbd33925d598
Author: P33M <P33M@github.com>
Date:   Thu Jul 24 21:24:03 2014 +0100

    usb: core: make overcurrent messages more prominent
    
    Hub overcurrent messages are more serious than "debug". Increase loglevel.

diff --git a/drivers/usb/core/hub.c b/drivers/usb/core/hub.c
index 50e8545..920e61b 100644
--- a/drivers/usb/core/hub.c
+++ b/drivers/usb/core/hub.c
@@ -4897,7 +4897,7 @@ static void port_event(struct usb_hub *hub, int port1)
 	if (portchange & USB_PORT_STAT_C_OVERCURRENT) {
 		u16 status = 0, unused;
 
-		dev_dbg(&port_dev->dev, "over-current change\n");
+		dev_notice(&port_dev->dev, "over-current change\n");
 		usb_clear_port_feature(hdev, port1,
 				USB_PORT_FEAT_C_OVER_CURRENT);
 		msleep(100);	/* Cool down */
